api/v1/helper.go

```diff
@@ -247,6 +248,13 @@ func NeedToUpdateSriov(ifaceSpec *Interface, ifaceStatus *InterfaceExt) bool {
 		log.V(2).Info("NeedToUpdateSriov(): NumVfs needs update", "desired", ifaceSpec.NumVfs, "current", ifaceStatus.NumVfs)
 		return true
 	}
+
+	var linkUp netlink.LinkOperState = netlink.OperUp
+	if ifaceStatus.LinkStatus != linkUp.String() {
```

Comment:
we should not have this check here.
there are cases where users use a down PF for communication between to VFs on the same PF.
they can configure `LinkState on` and just use the nic as a backbone for 2 DPDK applications passing traffic

Reply:
Hmm.. so basically users will overwrite what the operator does when it configures VFs [(i.e. set PF link up)](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/6e8a86125ffbe1e3539cf1a7451c86e90b229cbb/pkg/host/sriov.go#L454-L464) and rely on the operator to not move the PF link up?

This sounds hacky. Will bringing the PF up break their use case?